builder_requests.go

```diff
@@ -86,32 +86,32 @@ func (c *Client) BuilderTemplate(module *Module, template string) (*EditorStateR
 	return editorState, nil
 }
 
-// BuildFunction triggers a remote build for the given Module and function body. See also: Client.BuildFunctionString()
+// BuildPlugin triggers a remote build for the given Plugin and source code. See also: Client.BuildPluginString()
```

Comment:
```suggestion
// BuildPlugin triggers a remote build for the given plugin and source code. See also: Client.BuildPluginString()
```
Nit based on feedback in other repos.